### PR TITLE
{data}[intel/2018b] fix failing sanity check for NCL 6.6.2 due to missing dependencies (+ add easyconfig using foss/2018b)

### PR DESCRIPTION
**Pull request by casparvl**
(created using `eb --new-pr`)
Together with the EasyBlock for which I did a PR here ~~https://github.com/easybuilders/easybuild-easyblocks/pull/1862~~, this intel-based EasyConfig adds Bison to the build deps, to avoid a missing yacc error, and adds png and libjpeg-turbo as deps, to avoid the RPATH sanity check failing on those libraries not being found. In summary, it fixes https://github.com/easybuilders/easybuild-easyconfigs/issues/9332

### CODE REVIEW

**Comment by boegel** [comment]
@casparvl Can you take care of the missing dependencies and the `http://` URL in the `NCL` easyconfig?

File: easybuild/easyconfigs/n/NCL/NCL-6.6.2-intel-2018b.eb [hunk 1]
```diff
@@ -9,12 +9,19 @@ toolchainopts = {'cstd': 'c99', 'openmp': True, 'pic': True}
...
+    'NCL-6.4.0_fix-types.patch',
+]
 checksums = [
     'cad4ee47fbb744269146e64298f9efa206bc03e7b86671e9729d8986bb4bc30e',  # 6.6.2.tar.gz
     'f6dfaf95e5de9045745e122cb44f9c035f81fab92f5892991ddfe93945891c8f',  # NCL-6.4.0_fix-types.patch
 ]
 
+builddependencies = [
+    ('makedepend', '1.0.6'),
+    ('Bison', '3.0.5'),
```
Comment (boegel) [inline easybuild/easyconfigs/n/NCL/NCL-6.6.2-intel-2018b.eb]: @casparvl `flex` should be included as a build dependency as well

Comment (casparvl) [reply easybuild/easyconfigs/n/NCL/NCL-6.6.2-intel-2018b.eb]: Should be done now...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/f92d60f685ef04d703794214ab7c1c12 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2555.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/fab70627d68001c7761da393daa99943 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/e66b720543cd55d68a88cbe1b03c5273 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @casparvl!

**Pull request merged**